09/calculator_test.go

```diff
@@ -1,17 +1,64 @@
 package calculator_test
 
 import (
+	"calculator"
 	"testing"
 )
 
 func TestSubtract(t *testing.T) {
-	t.Fatal("Not implemented yet")
+	testCases := []struct {
+		a, b int
+		want int
+	}{
+		{a: 6, b: 7, want: -1},
+		{a: 9, b: 3, want: 6},
+		{a: 0, b: 0, want: 0},
+		{a: -6, b: -7, want: 1},
+		{a: -2, b: 5, want: -7},
+	}
+	for _, testCase := range testCases {
+		got := calculator.Subtract(testCase.a, testCase.b)
+		if got != testCase.want {
```

Comment:
Although it makes no difference to the code, I like to always compare my results in the order `want != got`. That way, it's more logical when you write something like:

```go
if !cmp.Equal(want, got) {
	t.Error(cmp.Diff(want, got))
}
```